src/main/java/skeltal/TaskList.java

```diff
@@ -0,0 +1,91 @@
+package skeltal;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
+    /**
+     * A method to add a Task object into the list.
+     * @param assignment A task object.
+     */
+    public static void addReply(Task assignment) {
+        tasks.add(assignment);
+        System.out.println("Got it. I've added this task");
+        System.out.println(assignment);
+        System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+    }
+
+    /**
+     * A method to list the Task objects currently in the list.
+     */
+    public static void listReply() {
+        for (Task task : tasks) {
+            System.out.println(task);
+        }
+    }
+
+    /**
+     * Returns the index position of the task in the list.
+     * @param task The Task object.
+     * @return The index position of the task in the list.
+     */
+    public static int getIndex(Task task) {
+        return tasks.indexOf(task);
+    }
+
+    /**
+     * Sets the Task at the index (index_1 - 1) in the TaskList to a "completed" state.
+     * @param index_1 The index of the task in the list + 1.
+     */
+    public static void done(String index_1) {
```

Comment:
It seems like you named this `index_1` since it represents `index + 1` rather than `index`, but is there another way to make this differentiation? Variable names should be in camelCase ([naming](https://se-education.org/guides/conventions/java/intermediate.html#naming)).